cmd/space.go

```diff
@@ -47,3 +47,26 @@ var rmSpaceCmd = &cobra.Command{
 		return registry.RemoveSpace(s)
 	},
 }
+
+var lsSpaceCmd = &cobra.Command{
+	Use:     "ls-spaces",
+	Short:   `List spaces`,
+	Long:    `List all configured spaces`,
+	PreRunE: compose(prepareRegistry, prepareSpaces),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		var spaceList []string
+
+		if physicalFlag || (!physicalFlag && !virtualFlag) {
```

Comment:
This could be simplified:
```suggestion
		if physicalFlag || !virtualFlag {
```

Reply:
right. Updated.